tests/functional/simple_seed/test_seed.py

```diff
@@ -1,19 +1,290 @@
+        self._build_relations_for_test(project)
+        self._check_relation_end_state(run_result=run_dbt(["seed"]), project=project, exists=True)
+        self._check_relation_end_state(
+            run_result=run_dbt(["seed", "--full-refresh"]), project=project, exists=True
+        )
+
+
+class TestSeedCustomSchema(SeedTestBase):
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql_file(project.test_data_dir / Path("seed_expected.sql"))
+
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {
+                "schema": "custom_schema",
+                "quote_columns": False,
+            },
+        }
+
+    def test_postgres_simple_seed_with_schema(self, project):
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 1/4 custom schema work; but unsure how to provide these lines with custom schema on seed_actual
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+        # this should truncate the seed_actual table, then re-insert
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 2/4
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+    def test_postgres_simple_seed_with_drop_and_schema(self):
+        results = run_dbt(["seed"])
+        assert len(results) == 1
+        # TODO: 3/4
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+        # this should drop the seed table, then re-create
+        results = run_dbt(["seed", "--full-refresh"])
+        results = run_dbt(["seed"])
+        # TODO: 4/4
+        # check_relations_equal(project.adapter, ["seed_actual", "seed_expected"])
+
+
+class TestSimpleSeedEnabledViaConfig(object):
+    @pytest.fixture(scope="class")
+    def seeds(self):
+        return {
+            "seed_enabled.csv": seeds__enabled_in_config,
+            "seed_disabled.csv": seeds__disabled_in_config,
+            "seed_tricky.csv": seeds__tricky,
+        }
+
+    @pytest.fixture(scope="class")
+    def project_config_update(self):
+        return {
+            "config-version": 2,
+            "seeds": {
+                "test": {"seed_enabled": {"enabled": True}, "seed_disabled": {"enabled": False}},
+                "quote_columns": False,
+            },
+        }
+
+    def test_postgres_simple_seed_with_disabled(self, project):
+        results = run_dbt(["seed"])
+        len(results) == 2
+        check_table_does_exist(project.adapter, "seed_enabled")
+        check_table_does_not_exist(project.adapter, "seed_disabled")
+        check_table_does_exist(project.adapter, "seed_tricky")
+        # TODO: 1/3 is there a better way to enforce cleanup
```

Comment:
So this is required cleanup. I'm guessing there is a better way to reset the database after every test_ when using the `def seeds` fixture in a test class.

Reply:
You can have your setup fixture do it by doing the setup, yield something and do the drop table after yield. [Examples](https://docs.pytest.org/en/6.2.x/fixture.html#yield-fixtures-recommended)

Reply:
We do now have project.drop_test_schema(). I haven't tested it for this use case, but it should work.

Reply:
I tried to get the `project.drop_test_schema()` to work but no dice. Digging into the code didn't reveal any answers. Opted for this instead:
```python
182     @pytest.fixture(scope="function")
183     def clear_test_schema(self, project):
184         yield
185         project.run_sql(f"drop schema if exists {project.test_schema} cascade")
186
187     def test_postgres_simple_seed_with_disabled(self, clear_test_schema, project):
```